multi-git-dashboard/src/components/Navbar.tsx

```diff
@@ -211,8 +211,7 @@ const Navbar: React.FC = () => {
   }, [router.pathname]);
 
   useEffect(() => {
-    const handleTabClose = (event: Event) => {
-      event.preventDefault();
+    const handleTabClose = () => {
```

Comment:
@auhc99 will this change affect the session logging?

Reply:
Not sure, Could you test it out?